rocketbelt/base/typography/_typography-cta.scss

```diff
@@ -4,9 +4,10 @@
   text-decoration: none;
   text-transform: uppercase;
   font-weight: font-weight(semibold);
+  line-height: 1.5;
```

Comment:
This should probably be `initial` in case the global `line-height` ever changes.